README.md

```diff
@@ -37,10 +37,10 @@ npm install
 pip install -r requirements.txt
 ```
   
-We require all contributor write docsting for other contributors easy to follow.  
-One of software development process is TDD. We highly recommended you *write test before code*  
-However, test is not require but recommended.  
-The philosophy is **you must know exactly the detailed behavior of your code doing**.  
+We require all contributors to write docsting for other contributors easy to follow.  
+One of the software development process is TDD(Test-driven Development). We highly recommended you *write at least one test before code*  
```

Comment:
Should we change `process` to `processes`?
`One of the software development processes is TDD(Test-driven Development). We highly recommended you *write at least one test before code*.`

Reply:
Good point! Resolved.